demisto_sdk/__main__.py

```diff
@@ -911,11 +911,9 @@ def id_set_command(**kwargs):
 
 
 # ====================== merge-id-sets ====================== #
-@main.command(name='merge-id-sets',
-              hidden=True,
-              short_help='Merge two id_sets')
-@click.help_option(
-    '-h', '--help'
```

Comment:
why remove the help?

Reply:
Because it is uneeded. It will use global help.
What going on here is adding -h as help. --help will always work.